src/tests/pointfree.js

```diff
@@ -0,0 +1,50 @@
+import { RuleTester } from "eslint"
+
+import rule from "rules/pointfree"
+
+const tests = {
+  valid: [
+    {
+      code: "const f = g",
```

Comment:
You can just supply a string as a shorthand for an object with `code` as the only key

Reply:
I get this error when I try that:
```
AssertionError [ERR_ASSERTION]: Should have no errors but had 1: [ { ruleId: null,
    fatal: true,
    severity: 2,
    message: 'Parsing error: The keyword \'const\' is reserved',
    line: 1,
    column: 1 } ]
```

Reply:
Ah right because we're "post-processing" these test cases in eg `tests.valid.forEach(...)`